vaapi/vaapidisplay_unittest.cpp

```diff
@@ -103,14 +103,9 @@ class VaapiDisplayTest : public ::testing::Test {
 
 #if __ENABLE_X11__
 VAAPI_DISPLAY_TEST(CreateInvalidHandleX11) {
-    FAIL(); // remove when fixed;
-
-    // FIXME: This causes segmentation fault.
-    // VaapiDisplay should be able to handle this gracefully
     NativeDisplay native = {1, NATIVE_DISPLAY_X11};
```

Comment:
Add `::testing::FLAGS_gtest_death_test_style = "threadsafe";` here to eliminate the gtest warning mentioned in your commit message